content/en/docs/concepts/overview/working-with-objects/field-selectors.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Field selectors
+weight: 60
+---
+
+_Field selectors_ enable you to select Kubernetes resources on the basis of the value of one or more resource fields. This `kubectl` command would select all pods for which the value of the `status.phase` field is `Running`:
+
+```shell
+$ kubectl get pods --field-selector status.phase=Running
+```
+
+{{< note >}}
+Field selectors are resource filters. By default, no filters are applied, meaning that all fields are selected for. This makes the following queries are equivalent:
+
+```shell
+$ kubectl get pods
+$ kubectl get pods --field-selector ""
+```
+{{< /note >}}
+
+## Supported operators
+
+The `!=` field selector operator is supported in addition to `=` (used in the example directly above). This `kubectl` command, for example, would select all Kubernetes Services that are not in the `default` namespace:
+
+```shell
+$ kubectl get services --field-selector metadata.namespace!=default
+```
+
+{{< warning >}}
+Please note that *only* the `=` and `!=` operators are supported as field selectors. [Label selector](../labels) syntax should not be taken as a general guide to field selector syntax. If you require more complex queries involving selectors like `in` or `notin`, you may be better served by label selectors.
+{{< /warning >}}
+
+## Chained selectors
+
+As with [label](../labels) and other selectors, field selectors can be chained together as a comma-separated list. This `kubectl` command would select all Pods for which the `status.phase` does not equal `Running` *and* the `spec.restartPolicy` field equals `Always`:
+
+```shell
+$ kubectl get pods --field-selector=status.phase!=Running,spec.restartPolicy=Always
+```
+
+## Multiple resource types
+
+Field selectors can be used to select across multiple resource types. This `kubectl` command would select all Statefulsets *and* Services that are not in the `default` namespace:
+
+```shell
+$ kubectl get statefulsets,services --field-selector metadata.namespace!=default
+```
+
+## Non-existent field errors
+
+If you attempt to select for a field that doesn't exist for the specified resource(s), you will see an error like this:
```

Comment:
It's not just whether the filed *exists*, IIUC most fields that exist are not usable in fieldSelector, only a few fields are supported, depending on object type?
E.g. The error message below lists only "metadata.name", "metadata.namespace" which are far from all ingress fields!

This was IMHO the main question in the issue that asked this to be documented.
- I'd mention that only some fields are supported up front at the start.
- Rename this section to "Supported fields" or "Non-supported fields" and change example to a field that exists but not supported for selection?

I don't know if maintaining a full list of supported fields in the docs is feasible.  If yes, probably not here but mark selectable fields in API docs?

Reply:
Unfortunately, maintaining a full list would likely be time prohibitive *unless* there were a way to autogenerate that list out of the Kubernetes codebase (and I doubt that that's currently possible).